pkg/operator/nfs/controller_test.go

```diff
@@ -110,16 +150,41 @@ func TestOnAdd(t *testing.T) {
 		},
 	}
 
+	nfsGaneshaConfig := `
+EXPORT {
+	Export_Id = 10;
+	Path = /test-claim;
+	Pseudo = /test-claim;
+	Protocols = 4;
+	Transports = TCP;
+	Sectype = sys;
+	Access_Type = RW;
+	Squash = none;
+	FSAL {
+		Name = VFS;
+	}
+}
+NFS_Core_Param
+{
+	fsid_device = true;
+}`
+	volMounts := []v1.VolumeMount{{Name: "test-claim", MountPath: "/test-claim"}, {Name: "nfs-ganesha-config", MountPath: "/nfs-ganesha/config"}}
+	onAdd(t, namespace, nfsserver, nfsGaneshaConfig, volMounts)
+
+}
+
+func onAdd(t *testing.T, namespace string, nfsServer *nfsv1alpha1.NFSServer, expectedConfig string, expectedVolumeMounts []v1.VolumeMount) {
```

Comment:
I'm hoping this provides an acceptable solution to the duplicated code issue - if not perhaps there is a better solution?